### ISSUE TITLE
Bug: Picklist - Secondary action button should not be used to check/uncheck box

### ISSUE THREAD

**Issue by annelfitz** [Esri/calcite-app-components#309]
### Summary
The user should only be able to use the space bar to check/uncheck the box when focused on the entire element. When focused specifically on the secondary action button, you should not be able to check/uncheck the box.

### Actual Behavior
When focused specifically on the secondary action button, you can check/uncheck the box by using the space bar

### Expected Behavior
When focused specifically on the secondary action button, you should not be able to check/uncheck the box.

### Reproduction Steps

1. https://esri.github.io/calcite-app-components/demos/calcite-pick-list.html
2. Tab through to focus on the secondary action button in the picklist
3. Hit the space bar - it will check/uncheck the checkbox

**Comment by annelfitz** [comment]
@pr3tori4n @driskull Looks good in chrome, but I'm noticing in firefox and safari that you cannot click on the secondary action. The focus does appear around it when you tab through, but physically clicking it removes the focus (if focus is already around it from tabbing). Otherwise, I am not seeing any indication that I actually clicked the button
![pick_list_no_click](https://user-images.githubusercontent.com/51344676/67504559-730abe80-f63e-11e9-90c2-ce9026bfe1fa.gif)

**Comment by pr3tori4n** [comment]
@annelfitz can you check the console? All the button does in the demo is log out the string 'clicked'.
That should validate the click event is firing. But it sounds like even if that's working the focus styling is not good? @asangma would you be able to take a look at that?

**Comment by driskull** [comment]
Focus styling issue. I think you need to set pointer-events to none on the svg element within.

**Comment by pr3tori4n** [comment]
I agree the styling of the secondary action states is too subtle. I brought this up with @asangma a while back and he said it's by design that the background colors on hover match, but I think it's worth a second look if multiple people are confused by it. @asangma can you weigh in when you get a chance?

**Comment by pr3tori4n** [comment]
@driskull the button in the `calcite-action` shadow dom is what receives focus when it's clicked. Doesn't seem there are any specific :focus styles on the button.
https://github.com/Esri/calcite-app-components/blob/master/src/components/calcite-action/calcite-action.scss#L6-L52

@asangma one solution you might consider is using rgba and adding a little transparency to the hover color, that way when they stack like the aciton on top of a list-item, it should get a little dark naturally.
https://jsfiddle.net/pr3tori4n/oesL13mk/1/
![hoverOpacity](https://user-images.githubusercontent.com/12614215/67976035-84217580-fbd2-11e9-8e74-d427629a27cb.gif)
